docs/blog/2019-03-26-what-the-jamstack-means-for-marketing/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: What the JAMstack means for marketing
+date: 2019-03-26
+author: Kym Ellis
+excerpt: "Learn what JAMstack is and the pros and cons JAMstack websites have from a marketing perspective!"
+tags:
+  - jamstack
+  - marketing
+  - headless cms
+canonicalLink: https://kymellis.co/jamstack-marketing-headless-cms/
+---
+
+There's an emerging trend in modern web development called the JAMstack. It's largely being discussed in developer circles, but I was lucky enough to work on a site relaunch project as a marketer where I got to see the JAMstack in action.
+
+It got me excited about the future because I've been in marketing for the best part of a decade and I've never witnessed a site that I've worked on go from zero to hero so quickly. There's often a disjoint in the decision making process when companies with resources (i.e. some money, developers and a marketing team) decide to launch or relaunch a marketing website. So in this post I'm going to summarise:
+
+- What the JAMstack is and what it means for tech-savvy marketers
+- An example of a JAMstack site that I worked on
+- The pros and cons from a marketing perspective
+
+## What is the JAMstack?
+
+JAM stands for JavaScript, APIs and Markup and if you're interested in reading more about the technical details you can do so [here](https://jamstack.org/), watch [this](https://www.youtube.com/watch?v=uWTMEDEPw8c) and read about the history of JAMstack [here](https://snipcart.com/blog/jamstack).
```

Comment:
I wish I'd seen this before it got merged, as these kinds of links should be more descriptive ("here", "this", "here" again).

Reply:
I'm happy to make any changes!